ompi/mpi/c/type_get_contents.c

```diff
@@ -65,6 +66,13 @@ int MPI_Type_get_contents(MPI_Datatype mtype,
         }
     }
 
+    /* Counts may exceed actual ones, we have no choice but to recompute them */
```

Comment:
Not sure I understand that comment: what are the `actual ones`? The count argument the user provided? Or the 

```suggestion
    /* Counts may exceed the user-provided counts, we have no choice but to recompute them */
```